### PR TITLE
add a link to the online manual in the about-page

### PR DESCRIPTION
**Pull request by t-m-z**

### CODE REVIEW

File: assets/i18n/de.json [hunk 1]
```diff
@@ -17,6 +17,9 @@
   "about_blog" : "Blog",
   "about_blog_url" : "https://blog.gcwizard.net/de/",
   "about_blog_url_text" : "blog.gcwizard.net",
+  "about_manual" : "Online-Handbuch",
+  "about_manual_url" : "https://blog.gcwizard.net/manual/",
```
Comment (S-Man42) [inline assets/i18n/de.json:21]: AFAIK there is a German link, isn't it?

Comment (Birdy2404) [reply assets/i18n/de.json:21]:
```suggestion
  "about_manual_url" : "https://blog.gcwizard.net/manual/de/",

```

File: assets/i18n/en.json [hunk 2]
```diff
@@ -17,6 +17,9 @@
   "about_blog" : "Blog",
   "about_blog_url" : "https://blog.gcwizard.net",
   "about_blog_url_text" : "blog.gcwizard.net",
+  "about_manual" : "Online manual",
+  "about_manual_url" : "https://blog.gcwizard.net/manual/",
```
Comment (Birdy2404) [inline assets/i18n/en.json:21]:
```suggestion
  "about_manual_url" : "https://blog.gcwizard.net/manual/en/",

```

File: assets/i18n/en.json [hunk 3]
```diff
@@ -17,6 +17,9 @@
   "about_blog" : "Blog",
   "about_blog_url" : "https://blog.gcwizard.net",
   "about_blog_url_text" : "blog.gcwizard.net",
+  "about_manual" : "Online manual",
+  "about_manual_url" : "https://blog.gcwizard.net/manual/",
+  "about_manual_url_text" : "blog.gcwizard.net/manual/",
```
Comment (Birdy2404) [inline assets/i18n/en.json:22]:
```suggestion
  "about_manual_url_text" : "blog.gcwizard.net/manual/en/",

```

File: assets/i18n/de.json [hunk 4]
```diff
@@ -17,6 +17,9 @@
   "about_blog" : "Blog",
   "about_blog_url" : "https://blog.gcwizard.net/de/",
   "about_blog_url_text" : "blog.gcwizard.net",
+  "about_manual" : "Online-Handbuch",
+  "about_manual_url" : "https://blog.gcwizard.net/manual/",
+  "about_manual_url_text" : "blog.gcwizard.net/manual/",
```
Comment (Birdy2404) [inline assets/i18n/de.json:22]:
```suggestion
  "about_manual_url_text" : "blog.gcwizard.net/manual/en/",

```

**Pull request merged**